### ISSUE TITLE
Discussion about the activity period

### ISSUE THREAD

**Issue by mariocao** [witnet/witnet-rust#1416]
Currently the activity period consensus constant may be leading to an under-representation of the network active population. Discuss if the activity period should be increased and which value should be set.

**Comment by drcpu-github** [comment]
I have watched the ARS size using my explorer over the past two weeks and noticed that is actively shrinking over time. It was ~400 nodes at the start, I saw it reduce to ~260 and is now at ~140 nodes. It seems to me that once nodes lose reputation, it is very hard to gain it back and become an ARS member again.

I have had multiple times where a node minted a block (and was thus active), but never got a chance to solve a data request, resulting in not gaining reputation, and thus not being able to solidify its position as an ARS member.

I am wondering if this is not a more systemic issue which will not be solved by just increasing the activity period. I am thinking that no matter how big you make the activity period, if the chances of entering the ARS are too small (or if you are simply not selected to solve a data request), the ARS will always stabilize on a very small size where reputation is just distributed amongst the same nodes.

Have there been any mechanical simulations performed of the ARS to check the long-term stability (this is obviously complicated to do faithfully)?
